compiler/rustc_error_messages/locales/en-US/hir_typeck.ftl

```diff
@@ -0,0 +1,8 @@
+hir_typeck_fru_note = this expression may have been misinterpreted as a `..` range expression
+hir_typeck_fru_expr = this expression does not end in a comma...
+hir_typeck_fru_expr2 = ... so this is interpreted as a `..` range expression, instead of functional record update syntax
```

Comment:
"functional record update syntax" is pretty heavy on jargon (and this is an error beginners will regularly see). Google search for it turns up nothing unless you add Rust, in which case you get this RFC: https://rust-lang.github.io/rfcs/2528-type-changing-struct-update-syntax.html

Reply:
```suggestion
hir_typeck_fru_expr2 = ... so this is interpreted as a `..` range expression, instead of functional record update syntax, used to set the remaining fields of a struct
```

Reply:
This part of the error message is already pretty long, and it doesn't wrap well -- if you're against the jargon, I would rather remove everything after `instead of`.

Reply:
I'm also not too keen on the duplication of "used to set the remaining fields of a struct" between the note and the suggestion

Reply:
Hmm, I see your point! I think it's better left as is then.

Reply:
Maybe "struct update syntax"? I've heard that quite often when talking about FRU.